ITEST/itest.storelogdata/test/org/openl/itest/RunStoreLogDataITest.java

```diff
@@ -123,15 +125,21 @@ public static void setUp() throws Exception {
 
     private void truncateTableIfExists(final String keyspace, final String table) {
         try {
-            EmbeddedCassandraServerHelper.getSession().execute("TRUNCATE " + keyspace + "." + table);
+            Optional<TableMetadata> tableMetadata = EmbeddedCassandraServerHelper.getSession()
+                    .getMetadata()
+                    .getKeyspace(KEYSPACE)
+                    .get().getTable(table);
+            if (tableMetadata.isPresent()) {
+                EmbeddedCassandraServerHelper.getSession().execute("TRUNCATE " + keyspace + "." + table);
+            }
         } catch (QueryExecutionException | InvalidQueryException ignored) {
         }
     }
 
-    private static void createKeyspaceIfNotExists(Session session,
-            String keyspaceName,
-            String replicationStrategy,
-            int replicationFactor) {
+    private static void createKeyspaceIfNotExists(CqlSession session,
+                                                  String keyspaceName,
+                                                  String replicationStrategy,
+                                                  int replicationFactor) {

```

Comment:
Please use **Eclipse Code Formatting** Plugin with https://github.com/openl-tablets/openl-tablets/blob/master/OpenL_Code_Style.xml and format it accordingly